ftplugin/go/commands.vim

```diff
@@ -85,6 +85,12 @@ if globpath(&rtp, 'plugin/ctrlp.vim') != ""
   command! -nargs=? -complete=dir GoDeclsDir call ctrlp#init(ctrlp#decls#cmd(1, <q-args>))
 endif
 
+" -- fzf
+if globpath(&rtp, 'plugin/fzf.vim') != ""
+  command! -nargs=? -complete=file GoDecls call fzf#decls#cmd(0, <q-args>)
+  command! -nargs=? -complete=dir GoDeclsDir call fzf#decls#cmd(1, <q-args>)
+endif
```

Comment:
We need to provide an options, such as `go_decls_mode` with the values of `ctrlp` and `fzf`. Otherwise if you have installed both `fzf` will override it.